### PR TITLE
Add verification test job

### PR DESCRIPTION
**Pull request by aishwaryabk**
To enable a job for running smoke tests of Verification-tests.

### CODE REVIEW

File: jobs/pipelines/powervs/ocp-4.12/daily-ocp4.12-powervs-verification-tests-script-sydney05-p9-min/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+                          sh '''
+                              cd ${WORKSPACE}/deploy
+                              scp -o 'StrictHostKeyChecking no' -i id_rsa ${WORKSPACE}/scripts/verification.sh root@${BASTION_IP}:
+                              ssh -o 'StrictHostKeyChecking no' -i id_rsa root@${BASTION_IP} "chmod 755 verification.sh;
+                                                                                              ~/verification.sh | tee ~/verification.log 2>&1 ;
+                                                                                              exit"
+                          '''
+                     }
+                     catch(err) {
+                         sleep 360
```
Comment (Shilpi-Das1) [inline jobs/pipelines/powervs/ocp-4.12/daily-ocp4.12-powervs-verification-tests-script-sydney05-p9-min/Jenkinsfile:166]: You can remove sleep

File: jobs/pipelines/powervs/ocp-4.12/daily-ocp4.12-powervs-verification-tests-script-sydney05-p9-min/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+            steps {
+                cloneRepo("git@github.ibm.com:redstack-power/ocp4_ansible_extras.git", "ansible_extra")
+            }
+        }
+        stage('Setup Common Environment Variables') {
+            steps {
+                setupCommonEnvironmentVariables()
+                setupClusterConfig("${CONFIG_TYPE}")
+                script {
+                    env.SYSTEM_TYPE = "e980"
```
Comment (Shilpi-Das1) [inline jobs/pipelines/powervs/ocp-4.12/daily-ocp4.12-powervs-verification-tests-script-sydney05-p9-min/Jenkinsfile:62]: You can remove this script tag and env.SYSTEM_TYPE = "e980"

**Comment by aishwaryabk** [comment]
Updated the changes. Thanks, @Shilpi-Das1

**Review by Shilpi-Das1 — APPROVED** [review]
/lgtm

**Review by sudeeshjohn — APPROVED** [review]
/lgtm

**Pull request merged**